packages/Telephony/Common/RegexAggregatorInput.cs

```diff
@@ -17,14 +18,14 @@ namespace Microsoft.Bot.Components.Telephony.Actions
     /// <summary>
     /// Aggregates input until it matches a regex pattern and then stores the result in an output property.
     /// </summary>
-    public class RegexAggregatorInput : Dialog
+    public abstract class RegexAggregatorInput : Dialog
```

Comment:
If this is abstract, that means that we don't expect it to be instantiable through declarative. In that case, can we make it internal, and then remove the Kind property?